tests/storage/nfs/test_nfs_sr.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from lib.common import wait_for, vm_image
+import time, subprocess
+
+# Requirements:
+# - one XCP-ng host >= 8.2 with an additional unused disk for the SR
+# - access to XCP-ng RPM repository from the host
+
+@pytest.mark.usefixtures("nfs_device_config")
```

Comment:
Is this line necessary?

Reply:
I don't think so, since you only have one test in the class and it already requires it. It's too bad that we (I) didn't document in what situations it's necessary for us. I think it has to do with test ordering but I'm not sure.